content.js

```diff
@@ -167,39 +167,52 @@ if (window.contentScriptInjected !== true) {
 
     function looksLikeForeignWord(word) {
         word = word.trim().toLowerCase();
+        if (word == "")
+            return false;
+        
+        const serbian_words_with_double_letters = [
```

Comment:
Maybe define this constant in a separate file or outside of this function since it makes it less readable. For more details check [this](https://refactoring.guru/smells/long-method) out.